frontend/src/views/CanteenEditor/CanteenForm.vue

```diff
@@ -199,23 +199,49 @@
           </v-radio-group>
         </v-col>
 
-        <v-col cols="12" md="6" :class="showDailyMealCount ? '' : 'grey--text text--darken-1'">
+        <v-col cols="12" md="4" :class="showDailyMealCount ? '' : 'grey--text text--darken-1'">
+          <label for="daily-meals" class="body-2 mb-2 d-block">
+            Couverts moyen par
+            <b>jour</b>
+            (convives sur place)
+          </label>
           <DsfrTextField
-            label="Couverts moyen par jour (convives sur place)"
+            id="daily-meals"
             hide-details="auto"
             :rules="showDailyMealCount ? [validators.greaterThanZero] : []"
             :disabled="!showDailyMealCount"
             :messages="showDailyMealCount ? [] : 'Concerne uniquement les cantines recevant des convives'"
             validate-on-blur
             v-model="canteen.dailyMealCount"
             prepend-icon="$restaurant-fill"
-            labelClasses="body-2 mb-2"
           />
         </v-col>
 
-        <v-col cols="12" md="6" :class="showSatelliteCanteensCount ? '' : 'grey--text text--darken-1'">
+        <v-col cols="12" md="4">
+          <label
+            for="yearly-meals"
+            class="body-2 d-block mb-2"
+            :class="{
+              'mb-lg-7': !showSatelliteCanteensCount,
```

Comment:
Minor remark : the alignment of the fields in large screens is a bit off when we have not selected the type of cantine. We could add the same margin classes to the field _Couverts moyen par jour (convives sur place)_ 

![image](https://user-images.githubusercontent.com/1225929/193784528-061fbaf8-3ea4-4d7b-9c72-e343065106a1.png)

Reply:
Bien vu - fait 7256789374